runtime/rehack/js/String_.js

```diff
@@ -1 +1 @@
-module.exports = require('String.js');
+module.exports = require('./String.js');
```

Comment:
How do you know the `String.js` will be here?

The `buildTest.sh` file has some examples of how to actually run the examples when compiled separately. For example:

```
NODE_PATH="${PWD}/rehack_tests/strings/:${PWD}/rehack_tests/stdlib/stdlib.js:${PWD}/runtime/rehack/js/" node -e 'require("Strings")'
```

Which puts the stdlib and other libraries in the node search path so that `require('String.js')` would be resolved correctly. But I'm not so sure `require('./String.js')` will work as accurately because it demands that `String_.js` be in the same dir as `String.js`. That would be fine if we had a system that copied these into the stdlib or something. We definitely need better tools for that.